sentry-android-okhttp/src/main/java/io/sentry/android/okhttp/SentryOkHttpInterceptor.kt

```diff
@@ -15,6 +15,8 @@ class SentryOkHttpInterceptor(
     private val beforeSpan: BeforeSpanCallback? = null
 ) : Interceptor {
 
+    constructor(beforeSpan: BeforeSpanCallback) : this(HubAdapter.getInstance(), beforeSpan)
```

Comment:
do you need to pass `HubAdapter.getInstance()` again as its the default value in the main ctor?

Reply:
Unless there is a trick that I don't know this is not possible
![image](https://user-images.githubusercontent.com/1357927/119800672-49321180-bedd-11eb-8ef2-912983956c65.png)

Reply:
mm yeah guess that's it then, coincides with the very same ctor.